posthog/management/commands/test/test_add_question_ids_to_surveys.py

```diff
@@ -0,0 +1,138 @@
+from django.core.management import call_command
+from django.db.models.signals import pre_save
+
+from posthog.models import Survey
+from posthog.models.feedback.survey import pre_save_survey
+from posthog.test.base import BaseTest
+from unittest.mock import patch
+
+
+class TestAddQuestionIdsToSurveys(BaseTest):
+    def setUp(self):
+        super().setUp()
+
+        # Disconnect the pre_save signal temporarily to create surveys without IDs
+        pre_save.disconnect(pre_save_survey, sender=Survey)
```

Comment:
curious why we have to do this.
i recall asking this before https://github.com/PostHog/posthog/pull/29209#discussion_r1971143857 and it was not a problem, what did it change?

Reply:
before it was being done in a migration, which already skips those signals by default. on the script we have to do this manually

Reply:
Only the test does that, why isnt the script not doing it then?

Reply:
it's not necessary to skip it - and there's also other `pre_signal` surveys that could be relevant for it.

for the test I have to make otherwise I can't really create surveys without ids to test if the script is behaving as expected

Reply:
I don't think we should run any signal here (such as pre_save_survey and friends) here, this should be exclusively setting and id, save and done.
If eg `update_survey_iterations`the operation will be way more expensive.
The other point is that we could likely mutate more fields that we don't want to (such as update_response_sampling_limits and more)

Reply:
actually, just learned the `bulk_update` (which we use to update) do not trigger pre_save of post_save signals for perf reasons, so no action needed here